Admin/AbstractAdmin.php

```diff
@@ -1978,6 +1985,11 @@ public function getBreadcrumbs($action)
      */
     public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
     {
+        @trigger_error(
+            'The '.__METHOD__.' method is deprecated since version 3.x and will be removed in 4.0 .',
```

Comment:
I hesitated, but other wise it looks like a 3 figures version number missing the last number.